Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetBillingCollectionUITests.swift

```diff
@@ -0,0 +1,163 @@
+//
+//  PaymentSheetBillingCollectionUITests.swift
+//  PaymentSheetUITest
+//
+//  Created by Eduardo Urias on 2/23/23.
+//
+
+import XCTest
+
+final class PaymentSheetBillingCollectionUITests: XCTestCase {
+    var app: XCUIApplication!
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        // Put setup code here. This method is called before the invocation of each test method in the class.
+
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+
+        app = XCUIApplication()
+        app.launchEnvironment = ["UITesting": "true"]
+        app.launch()
+    }
+
+    func testCard_AutomaticFields_NoDefaults() throws {
+        loadPlayground(
+            app,
+            settings: [
+                "customer_mode": "guestmode",
+                "currency": "USD",
+                "merchant_country_code": "US",
+                "apple_pay": "off",
+                "automatic_payment_methods": "off",
+                "link": "off",
+                "attach_defaults": "off",
+                "collect_name": "auto",
+                "collect_email": "auto",
+                "collect_phone": "auto",
+                "collect_address": "auto",
+            ]
+        )
+        app.buttons["Checkout (Complete)"].tap()
+
+        guard let card = scroll(collectionView: app.collectionViews.firstMatch, toFindCellWithId: "card") else {
+            XCTFail()
```

Comment:
Should we fail with a message to make the error more diagnosable, or does Xcode already give us enough diagnostic info here?

Reply:
Xcode will fail on this line, so that should be enough context, especially given that this is only meant to safely unwrap a value that should always exist, i.e. we're not testing whether there is a card option, there should be another test for that, we're just trying to get that option and tap it, but the result is an optional so it needs to be unwrapped.